src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTest.cs

```diff
@@ -13,7 +13,7 @@ namespace System.Diagnostics.ProcessTests
     public partial class ProcessTest : IDisposable
     {
         private const int WaitInMS = 100 * 1000; 
-        private const string CoreRunName = "corerun";
+        private const string CoreRunName = "corerun.exe";
```

Comment:
MainModule returns the filename with the extension and so I updated this as it works on windows. I did not realize Unix won't work with this. I will make a commit that changes MainModule test instead. (In general, it looks like our CI system only does windows validation, how is one suppose to ensure that everything works cross-plat?)